lib/mimic/module.ex

```diff
@@ -52,12 +52,12 @@ defmodule Mimic.Module do
 
     case :compile.forms(forms, compiler_options(module)) do
       {:ok, module_name, binary} ->
-        load_binary(module_name, binary)
+        load_binary(module_name, binary, Cover.enabled?(module))
         binary
 
       {:ok, module_name, binary, _warnings} ->
-        load_binary(module_name, binary)
-        Binary
```

Comment:
i assume this was a typo. but return val is never used for this fun

Reply:
Yup thanks!